test/io.jl

```diff
@@ -1,28 +1,28 @@
 module TestIO
     using Test, DataFrames, CategoricalArrays, Dates
-    using LaTeXStrings
+    # using LaTeXStrings
```

Comment:
Could you add a comment explaining why this and the corresponding tests are commented out?

Reply:
Added.